about-osm-community/_posts/0000-01-04-consumers.md

```diff
@@ -62,8 +62,10 @@ Here is a non-exhaustive list of some big organizations that use OpenStreetMap,
 * Clinton Health Access Initiative
 * Concern Worldwide
 * Gates Foundation
-* International Red Cross Red Crescent Societies (IFRC) and the International Committee of Red Cross (ICRC)
```

Comment:
"International Red Cross Red Crescent Societies" is not the correct spelled out for IFRC